recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: 7ef4694e1345913182126b219aaa4a0047e191af414256da6772cf249571b961
 
 build:
-  number: 1
+  number: 2
```

Comment:
it also fixes the specifier on backports.shutil_get_terminal_size, which should only be for py2. That means the py3 packages will have fixed dependencies on all platforms.